backend/app/models/__init__.py

```
@@ -108,4 +110,4 @@
     UserUpdateMe,
     UsersPublic,
     UpdatePassword,
-)
+)
```

Ruff (0.13.3)

No newline at end of file

Add trailing newline

(W292)